src/SaatchiArt/BentoBoxDDD/Adapters/Repositories/ArtworkRepositoryMysqlImpl.php

```diff
@@ -2,30 +2,24 @@
 
 declare(strict_types=1);
 
-namespace SaatchiArt\BentoBoxDDD\SecondaryAdapters\Repositories;
+namespace SaatchiArt\BentoBoxDDD\Adapters\Repositories;
 
-use Illuminate\Database\ConnectionInterface as Database;
+use SaatchiArt\BentoBoxDDD\Domain\Repositories\ArtworkRepository;
 use SaatchiArt\BentoBoxDDD\Entities\Artworks\ArtworkEntity;
 use SaatchiArt\BentoBoxDDD\Entities\Artworks\ArtworkImageValueObject;
-use SaatchiArt\BentoBoxDDD\Events\ArtworkUpdatedEvent;
-use SaatchiArt\BentoBoxDDD\Services\UserActions\SecondaryAdapters\Repositories\ArtworkRepositoryInterface;
 
-final class ArtworkRepository implements ArtworkRepositoryInterface
+final class ArtworkRepositoryMysqlImpl extends AbstractSingleConnectionRepository implements ArtworkRepository
```

Comment:
I was thinking `MysqlImpl` because it's a specific implementation of the interface. 
If it were to get switched to another database implementation, like SqlLite, then probably copy paste, rename it, and adjust it to work with SqlLite (if need be).